src/components/related-assets/related-assets.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Link as RouterLink } from "react-router-dom";
+
+import { Link, LinkBox, LinkOverlay } from "@mtfh/common/lib/components";
+import { SearchCard } from "@mtfh/search";
+
+import { RelatedAsset } from "views/related-assets-view/utils";
+
+export interface RelatedAssetsProps {
+  assetType: string;
+  relatedAssets: RelatedAsset[];
+}
+
+export const RelatedAssets = ({ assetType, relatedAssets }: RelatedAssetsProps) => {
+  const getAssetTypeHeading = () => {
+    return assetType.charAt(assetType.length - 1).toLowerCase() === "s"
+      ? assetType
+      : `${assetType}s`;
+  };
+
+  const renderRelatedAssets = () => {
+    return relatedAssets.map((relatedAsset) => {
+      return (
+        <LinkBox key={relatedAsset.id} data-testid="related-asset">
+          <SearchCard>
+            <LinkOverlay>
+              <Link
+                className="mtfh-search-tenure__title"
+                as={RouterLink}
+                to={`/property/${relatedAsset.id}`}
+                variant="text-colour"
+              >
+                {relatedAsset.name}
+              </Link>
+            </LinkOverlay>
+          </SearchCard>
+        </LinkBox>
+      );
+    });
+  };
```

Comment:
Can this be a component?

Reply:
Introduced RelatedAsset component

Reply:
React is designed around components, it is intended for components to be the building block of the UI. Using functions that return JSX breaks this pattern, and over time will make the code harder to maintain.

You should either create a RelatedAsset component (it can be in the same file), or just include the jsx directly in the return

Reply:
I'm confused, my previous comment says "Introduced RelatedAsset component" and I did exactly that yesterday, did you review the changes?